aptos-move/framework/aptos-experimental/sources/daa_siws_phantom.move

```diff
@@ -0,0 +1,255 @@
+/// Derivable account abstraction that verifies a message signed by
+/// SIWS from a Phantom wallet.
+module aptos_experimental::daa_siws_phantom {
+    use aptos_framework::auth_data::AbstractionAuthData;
+    use aptos_std::ed25519::{
+        Self,
+        new_signature_from_bytes,
+        new_unvalidated_public_key_from_bytes,
+    };
+    use std::chain_id;
+    use std::error;
+    use std::option::{Self, Option};
+    use std::string_utils;
+    use std::transaction_context::{Self, EntryFunctionPayload};
+    use std::vector;
+
+    /// Signature failed to verify.
+    const EINVALID_SIGNATURE: u64 = 1;
+    /// Non base58 character found in public key.
+    const EINVALID_BASE_58_PUBLIC_KEY: u64 = 2;
+    /// Entry function payload is missing.
+    const EMISSING_ENTRY_FUNCTION_PAYLOAD: u64 = 3;
+
+    const BASE_58_ALPHABET: vector<u8> = b"123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz";
```

Comment:
There is no capital `I`, right?

Maybe add a comment to clarify: 
```
// a 58-character alphabet consisting of numbers (1-9) and almost all (A-Z, a-z) letters,
// excluding 0, O, I, and l to avoid confusion between similar-looking characters.
```